src/script/script.cpp

```diff
@@ -15,144 +15,157 @@ const char* GetOpName(opcodetype opcode)
+	case OP_NOT                    : return "OP_NOT";
+	case OP_0NOTEQUAL              : return "OP_0NOTEQUAL";
+	case OP_ADD                    : return "OP_ADD";
+	case OP_SUB                    : return "OP_SUB";
+	case OP_MUL                    : return "OP_MUL";
+	case OP_DIV                    : return "OP_DIV";
+	case OP_MOD                    : return "OP_MOD";
+	case OP_LSHIFT                 : return "OP_LSHIFT";
+	case OP_RSHIFT                 : return "OP_RSHIFT";
+	case OP_BOOLAND                : return "OP_BOOLAND";
+	case OP_BOOLOR                 : return "OP_BOOLOR";
+	case OP_NUMEQUAL               : return "OP_NUMEQUAL";
+	case OP_NUMEQUALVERIFY         : return "OP_NUMEQUALVERIFY";
+	case OP_NUMNOTEQUAL            : return "OP_NUMNOTEQUAL";
+	case OP_LESSTHAN               : return "OP_LESSTHAN";
+	case OP_GREATERTHAN            : return "OP_GREATERTHAN";
+	case OP_LESSTHANOREQUAL        : return "OP_LESSTHANOREQUAL";
+	case OP_GREATERTHANOREQUAL     : return "OP_GREATERTHANOREQUAL";
+	case OP_MIN                    : return "OP_MIN";
+	case OP_MAX                    : return "OP_MAX";
+	case OP_WITHIN                 : return "OP_WITHIN";
+
+	// crypto
+	case OP_RIPEMD160              : return "OP_RIPEMD160";
+	case OP_SHA1                   : return "OP_SHA1";
+	case OP_SHA256                 : return "OP_SHA256";
+	case OP_HASH160                : return "OP_HASH160";
+	case OP_HASH256                : return "OP_HASH256";
+	case OP_CODESEPARATOR          : return "OP_CODESEPARATOR";
+	case OP_CHECKSIG               : return "OP_CHECKSIG";
+	case OP_CHECKSIGVERIFY         : return "OP_CHECKSIGVERIFY";
+	case OP_CHECKMULTISIG          : return "OP_CHECKMULTISIG";
+	case OP_CHECKMULTISIGVERIFY    : return "OP_CHECKMULTISIGVERIFY";
+
+	// expansion
+	case OP_NOP1                   : return "OP_NOP1";                  // OP_NOP1
+	case OP_CHECKLOCKTIMEVERIFY    : return "OP_CHECKLOCKTIMEVERIFY";   // OP_NOP2
+	case OP_NOP3                   : return "OP_NOP3";                  // OP_NOP3
+	case OP_NOP4                   : return "OP_NOP4";                  // OP_NOP4
+	case OP_NOP5                   : return "OP_NOP5";                  // OP_NOP5
+	case OP_NOP6                   : return "OP_NOP6";                  // OP_NOP6
+	case OP_NOP7                   : return "OP_NOP7";                  // OP_NOP7
+	case OP_NOP8                   : return "OP_NOP8";                  // OP_NOP8
+	case OP_NOP9                   : return "OP_NOP9";                  // OP_NOP9
+	case OP_NOP10                  : return "OP_NOP10";                 // OP_NOP10
+
+	// zerocoin
+	case OP_ZEROCOINMINT           : return "OP_ZEROCOINMINT";
+	case OP_ZEROCOINSPEND          : return "OP_ZEROCOINSPEND";
+	case OP_ZEROCOINPUBLICSPEND    : return "OP_ZEROCOINPUBLICSPEND";
+
+	// memory store
+	case OP_MLOAD				   : return "OP_MLOAD";
+	case OP_MSTORE				   : return "OP_MSTORE";
+	case OP_MSTORE8				   : return "OP_MSTORE8";
```

Comment:
lets do only the OP_MSTORE for now, we add the 8 bit version later if we really need it

Reply:
:+1: